cloudesire-api-client-dto/src/main/java/com/cloudesire/platform/apiclient/dto/model/dto/ProceedsTotalsDTO.java

```diff
@@ -4,21 +4,25 @@
 
 import java.math.BigDecimal;
 
-public class ResellingTotalsDTO extends DTO
+/**
+ * @deprecated by headers in GET /proceedsReport
+ */
+@Deprecated
+public class ProceedsTotalsDTO extends DTO
```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Do not forget to remove this deprecated code someday. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.cloudesire.com/coding_rules#rule_key=squid%3AS1133)